templates/nifi.service.j2

```diff
@@ -9,6 +9,9 @@ User={{ nifi_user }}
 Group={{ nifi_group }}
 ExecStart={{ nifi_config_dirs.home }}/bin/nifi.sh start
 ExecStop={{ nifi_config_dirs.home }}/bin/nifi.sh stop
+{% if restart_nifi | default(false) %}
```

Comment:
add check if var contains one of 
no always	on-success	on-failure	on-abnormal	on-abort	on-watchdog